pkg/larker/loader/loader.go

```diff
@@ -65,11 +71,28 @@ func (loader *Loader) LoadFunc() func(thread *starlark.Thread, module string) (s
 
 		// A special case for loading Cirrus-provided builtins (e.g. load("cirrus", "fs"))
 		if module == "cirrus" {
+			starlibModules, err := loadStarlibModules()
+			if err != nil {
+				return nil, err
+			}
+
+			starlarkEnv := starlark.NewDict(len(loader.env))
+			for key, value := range loader.env {
+				if err := starlarkEnv.SetKey(starlark.String(key), starlark.String(value)); err != nil {
+					return nil, err
+				}
+			}
+
 			return starlark.StringDict{
 				"fs": &starlarkstruct.Module{
 					Name:    "fs",
 					Members: builtin.FS(loader.ctx, loader.fs),
 				},
+				"env":  starlarkEnv,
```

Comment:
Seems we can rename `starlibModules` to `builtinModules` and then do something like:

```python
builtinModules["fs"] = ...
builtinModules["env"] = ...
return builtinModules
```

This way there is no need for duplicated `"http"`, `"hash"`, etc. What do you think?

Reply:
See https://github.com/cirruslabs/cirrus-cli/pull/96/commits/9645b0411b06fbfe0062c77cc16d3f023fae8734.